src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -166,16 +165,19 @@ const connectCurrentRefinements: CurrentRefinementsConnector = (
         );
       },
 
-      render({ results, helper, state, createURL, instantSearchInstance }) {
-        const items = transformItems(
-          getItems({
-            results,
-            state,
-            helper,
-            includedAttributes,
-            excludedAttributes,
-          })
-        );
+      render({ scopedResults, helper, createURL, instantSearchInstance }) {
+        const items = scopedResults.reduce<Item[]>((results, scopedResult) => {
+          return results.concat(
+            transformItems(
+              getItems({
+                results: scopedResult.results,
```

Comment:
simply  calling it like this will not remove duplicates, as shown in the test, we need to make sure we fetch "top-down" and ignore ones we already have. Then we need to be careful, because they could have different values, in which case there would still be two.

Reply:
See https://github.com/algolia/instantsearch.js/pull/4012#discussion_r312417432.

Reply:
Fixed f994a628ca7d29896e6cd25c2219a1c6ec4eff5f.